jsonschema-core.xml

```diff
@@ -1845,6 +1835,82 @@
                 </section>
             </section>
 
+            <section title="Compound Documents">
+                <t>
+                    A Compound Schema Document is defined as JSON document (sometimes called a "bundled" schema)
+                    which has multiple embedded JSON Schema Resources bundled into the same document to
+                    ease transportation.
+                </t>
+                <t>
+                    Each embedded Schema Resource MUST be treated as an individual Schema Resource, following standard
+                    schema loading and processing requirements, including determining voacbulary support.
+                </t>
+                <section title="Bundling">
+                    <t>
+                        The bundling process for creating a Compound Schema Document is defined as taking
+                        references (such as "$ref") to an external Schema Resource and embedding the referenced
+                        Schema Resources within the referreing document.
+                    </t>
+                    <t>
+                        Each embedded JSON Schema Resource MUST identify itself with an absolute URI using the "$id" keyword,
```

Comment:
Is this referring to embedded resources in the resulting schema (after bundling has been performed), or the original source resources, or both?  My earlier questions about this being required vs. recommended still apply -- it need not be a requirement that referenced resources use absolute URIs if it is acceptable to have rewritten $refs in the result (that is, a new schema be produced that has different canonical uris at various subschemas than in the original, and the user-application understands that the locations in resulting errors or annotations will be different than had the original non-bundled schemas been used).

Reply:
I don't think this is exclusive to canonical URIs. I'm going with a slightly altered version of your suggestion. Thanks!